docs/topics/proc_show-debug-logging-local-nodejs.adoc

```diff
@@ -0,0 +1,32 @@
+= Show Debug Logging on Localhost
+
+Use the `DEBUG` environment variable when starting your application to enable debug logging.
+
+.Prerequisites 
+* An application with debug logging.
+
+.Procedure
+. Set the `DEBUG` environment variable to enable debug logging in your application.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ DEBUG=mybooster npm start
+----
++
+The `DEBUG` environment variable uses link:https://www.npmjs.com/package/debug#wildcards[wildcards] to filter debugging messages.
```

Comment:
I think it's the `debug` package who _uses_ wildcards. The `DEBUG` environment variable _can include_ wildcards :-)

Reply:
Slightly more refined, "The `DEBUG` environment variable _specifies_ which debug statements to print, and may include wildcards."